### DISCUSSION TITLE
it's possible add translations on head html meta tag?

### DISCUSSION THREAD

**Discussion by Cleysonlb** [vinissimus/next-translate#412] [General]
Hey! it's possible add translations on head html meta tag?

example:
```
import React from 'react';
import Head from 'next/head';
import getConfig from 'next/config';
import useTranslation from 'next-translate/useTranslation';
import Home from '../components/pages/Home';

const HomePage = props => {
  const { t } = useTranslation('translation');
  const { publicRuntimeConfig } = getConfig();
  const { siteUrl } = publicRuntimeConfig;

  return (
    <>
      <Head>
        <title>Website</title>
        <meta name="robots" content="index,follow" />
        <meta name="description" content={t('home.headerContent')} />
        <link rel="canonical" href={siteUrl} />
      </Head>
      <Home {...props} />
    </>
  );
};

export default HomePage;
```

![image](https://user-images.githubusercontent.com/7593949/102524465-d1cb9c80-4077-11eb-9341-1495daf4b4b2.png)

**Comment by aralroca** [comment]
@Cleysonlb Yes, it's possible. In the repo examples we are using it:

https://github.com/vinissimus/next-translate/blob/1.0.0/examples/without-loader/components/header.js#L21

If you use the same translations outside the Head, does it work? It looks like it's missing the translation or the page is not downloading the [namespace](https://github.com/vinissimus/next-translate#how-are-translations-loaded).
